### ISSUE TITLE
Setoken (com.apple.CryptoTokenKit.setoken) crashing

### ISSUE THREAD

**Issue by Zikofurito** [PoomSmart/EmojiPort-PE#7]
Crashlog https://pastebin.com/7GBqR1iU

**Comment by PoomSmart** [comment]
How can you reproduce this issue?

**Comment by Zikofurito** [comment]
> How can you reproduce this issue?

I currently don’t know how to reproduce the crash

**Comment by yur1xpp** [comment]
I can reproduce this in Supercharge app

1. create simple tweak
2. Tap install
3. Notice the “the operation couldn’t be completed” alert, where setoken crashed simultaneously. (Doesn’t happen with beta3)


I also having the same crash on scfileops

```
Process: scfileops
Bundle id: (null)
Device: iPad Pro (12.9-inch) (4th generation), iOS 14.3

Exception type: EXC_SOFTWARE (SIGABRT)
Exception subtype: EXC_UNIX_ABORT
Exception codes: 0x0000000000010002, 0x0000000000000000
Culprit: EmojiPortPE.dylib

Triggered by thread: 0
Thread name: Dispatch queue: com.apple.main-thread
Call stack:
0   libsystem_kernel.dylib        	0x00000001c397e414 0x1c3956000 + 164884     	// __pthread_kill
1   libsystem_pthread.dylib       	0x00000001e0db6b40 0x1e0db4000 + 11072      	// pthread_kill
2   libsystem_c.dylib             	0x000000019f6b3b74 0x19f63d000 + 486260     	// abort
3   libsubstitute.dylib           	0x00000001009d3450 0x1009c0000 + 78928      	// SubHookFunction.cold.2
4   libsubstitute.dylib           	0x00000001009d0790 0x1009c0000 + 67472      	// SubHookFunction
5   EmojiAttributes.dylib         	0x0000000100dfdf64 0x100df8000 + 24420      	// _logosLocalCtor_29b31af0(int, char**, char**)
6   dyld                          	0x0000000100402888 0x1003e8000 + 108680     	// ImageLoaderMachO::doModInitFunctions(ImageLoader::LinkContext const&)
7   dyld                          	0x0000000100402b10 0x1003e8000 + 109328     	// ImageLoaderMachO::doInitialization(ImageLoader::LinkContext const&)
8   dyld                          	0x00000001003fcc68 0x1003e8000 + 85096      	// ImageLoader::recursiveInitialization(ImageLoader::LinkContext const&, unsigned int, char const*, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
9   dyld                          	0x00000001003fabb8 0x1003e8000 + 76728      	// ImageLoader::processInitializers(ImageLoader::LinkContext const&, unsigned int, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
10  dyld                          	0x00000001003fac84 0x1003e8000 + 76932      	// ImageLoader::runInitializers(ImageLoader::LinkContext const&, ImageLoader::InitializerTimingList&)
11  dyld                          	0x00000001003ee4d0 0x1003e8000 + 25808      	// dyld::runInitializers(ImageLoader*)
12  dyld                          	0x00000001003f6334 0x1003e8000 + 58164      	// dlopen_internal
13  libdyld.dylib                 	0x0000000195f4fd74 0x195f4a000 + 23924      	// dlopen_internal(char const*, int, void*)
14  EmojiPortPE.dylib             	0x0000000100defd9c 0x100de8000 + 32156      	// func_7b84
15  dyld                          	0x000000010040270c 0x1003e8000 + 108300     	// ImageLoaderMachO::doModInitFunctions(ImageLoader::LinkContext const&)
16  dyld                          	0x0000000100402b10 0x1003e8000 + 109328     	// ImageLoaderMachO::doInitialization(ImageLoader::LinkContext const&)
17  dyld                          	0x00000001003fcc68 0x1003e8000 + 85096      	// ImageLoader::recursiveInitialization(ImageLoader::LinkContext const&, unsigned int, char const*, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
18  dyld                          	0x00000001003fabb8 0x1003e8000 + 76728      	// ImageLoader::processInitializers(ImageLoader::LinkContext const&, unsigned int, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
19  dyld                          	0x00000001003fac84 0x1003e8000 + 76932      	// ImageLoader::runInitializers(ImageLoader::LinkContext const&, ImageLoader::InitializerTimingList&)
20  dyld                          	0x00000001003ee4d0 0x1003e8000 + 25808      	// dyld::runInitializers(ImageLoader*)
21  dyld                          	0x00000001003f6334 0x1003e8000 + 58164      	// dlopen_internal
22  libdyld.dylib                 	0x0000000195f4fd74 0x195f4a000 + 23924      	// dlopen_internal(char const*, int, void*)
23  substitute-loader.dylib       	0x0000000100a7f5e0 0x1009ec000 + 603616     
24  substitute-loader.dylib       	0x0000000100c3e240 0x1009ec000 + 2433600    
25  substitute-loader.dylib       	0x0000000100c3e264 0x1009ec000 + 2433636    
26  substitute-loader.dylib       	0x0000000100c3e288 0x1009ec000 + 2433672    
27  substitute-loader.dylib       	0x0000000100c0caa8 0x1009ec000 + 2230952    
28  substitute-loader.dylib       	0x0000000100b61c00 0x1009ec000 + 1530880    
29  substitute-loader.dylib       	0x0000000100b60a48 0x1009ec000 + 1526344    
30  substitute-loader.dylib       	0x0000000100b992ec 0x1009ec000 + 1757932    
31  substitute-loader.dylib       	0x0000000100b655b4 0x1009ec000 + 1545652    
32  dyld                          	0x000000010040270c 0x1003e8000 + 108300     	// ImageLoaderMachO::doModInitFunctions(ImageLoader::LinkContext const&)
33  dyld                          	0x0000000100402b10 0x1003e8000 + 109328     	// ImageLoaderMachO::doInitialization(ImageLoader::LinkContext const&)
34  dyld                          	0x00000001003fcc68 0x1003e8000 + 85096      	// ImageLoader::recursiveInitialization(ImageLoader::LinkContext const&, unsigned int, char const*, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
35  dyld                          	0x00000001003fabb8 0x1003e8000 + 76728      	// ImageLoader::processInitializers(ImageLoader::LinkContext const&, unsigned int, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
36  dyld                          	0x00000001003fac84 0x1003e8000 + 76932      	// ImageLoader::runInitializers(ImageLoader::LinkContext const&, ImageLoader::InitializerTimingList&)
37  dyld                          	0x00000001003ee4d0 0x1003e8000 + 25808      	// dyld::runInitializers(ImageLoader*)
38  dyld                          	0x00000001003f6334 0x1003e8000 + 58164      	// dlopen_internal
39  libdyld.dylib                 	0x0000000195f4fd74 0x195f4a000 + 23924      	// dlopen_internal(char const*, int, void*)
40  substitute-inserter.dylib     	0x00000001005c123c 0x1004d8000 + 954940     	// func_e8f2c
41  substitute-inserter.dylib     	0x00000001008fa0a0 0x1004d8000 + 4333728    	// func_422090
42  substitute-inserter.dylib     	0x00000001008b5340 0x1004d8000 + 4051776    	// func_3d1760
43  dyld                          	0x000000010040270c 0x1003e8000 + 108300     	// ImageLoaderMachO::doModInitFunctions(ImageLoader::LinkContext const&)
44  dyld                          	0x0000000100402b10 0x1003e8000 + 109328     	// ImageLoaderMachO::doInitialization(ImageLoader::LinkContext const&)
45  dyld                          	0x00000001003fcc68 0x1003e8000 + 85096      	// ImageLoader::recursiveInitialization(ImageLoader::LinkContext const&, unsigned int, char const*, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
46  dyld                          	0x00000001003fabb8 0x1003e8000 + 76728      	// ImageLoader::processInitializers(ImageLoader::LinkContext const&, unsigned int, ImageLoader::InitializerTimingList&, ImageLoader::UninitedUpwards&)
47  dyld                          	0x00000001003fac84 0x1003e8000 + 76932      	// ImageLoader::runInitializers(ImageLoader::LinkContext const&, ImageLoader::InitializerTimingList&)
48  dyld                          	0x00000001003ea7a4 0x1003e8000 + 10148      	// dyld::initializeMainExecutable()
49  dyld                          	0x00000001003efd88 0x1003e8000 + 32136      	// dyld::_main(macho_header const*, unsigned long, int, char const**, char const**, char const**, unsigned long*)
50  dyld                          	0x00000001003e9258 0x1003e8000 + 4696       	// dyldbootstrap::start(dyld3::MachOLoaded const*, int, char const**, dyld3::MachOLoaded const*, unsigned long*)
51  dyld                          	0x00000001003e9038 0x1003e8000 + 4152       	// _dyld_start
```

**Comment by PoomSmart** [comment]
Fixed in 1.2.3
